ui/imports/shared/popups/walletconnect/ConnectDAppModal.qml

```diff
@@ -400,7 +218,17 @@ StatusDialog {
 
         readonly property var dappChains: ListModel {}
 
-        property int connectionStatus: notConnectedStatus
-        property bool afterTwoSecondsFromStatus: false
+        property int connectionStatus: root.notConnectedStatus
+        readonly property bool connectionSuccessful: d.connectionStatus === root.connectionSuccessfulStatus
+        readonly property bool connectionFailed: d.connectionStatus === root.connectionFailedStatus
+        readonly property bool connectionAttempted: d.connectionStatus !== root.notConnectedStatus
+
+        function syncDAppChainsModel() {
```

Comment:
I'm wondering why are we using a manual sync and not a proxy model.

Reply:
Good point, thanks! I've mindlessly kept the dappChains in the API. This component doesn't need to know about it. Removed it from the API and replaced with a SFPM filtering the flatNetworks to show only the supported chains.